libs/sdk-ui-kit/src/Header/typings.ts

```diff
@@ -49,6 +49,9 @@ export interface IAppHeaderProps {
 
     helpRedirectUrl?: string;
     theme?: ITheme;
+
+    isDisplayUpsellButton?: boolean;
+    onUpsellButtonClick?: (e?: React.MouseEvent) => void;
```

Comment:
The parameter `e` should not be optional, shouldn't it?

Reply:
I tested it, we can remove it thanks